app/src/main/java/org/mozilla/rocket/deeplink/DeepLinkType.kt

```diff
@@ -172,9 +171,23 @@ enum class DeepLinkType {
             return NOT_SUPPORT
         }
 
+        private fun isContentLink(uri: URI, path: String, hasQuery: Boolean): Boolean {
+            return isContentLink(uri) && path == uri.path && if (hasQuery) {
+                !uri.query.isNullOrEmpty()
+            } else {
+                uri.query.isNullOrEmpty()
+            }
+        }
+
         private fun isContentLink(uri: URI) =
                 isDeepLink(uri) && DeepLinkConstants.HOST_CONTENT == uri.host
 
+        private fun isCommand(uri: URI, command: String) = isCommand(uri) &&
+                uri.getParam(DeepLinkConstants.COMMAND_PARAM_KEY) == command
+
+        private fun isCommand(uri: URI) =
```

Comment:
Maybe we could name it `isCommandUri` to be differentiated from the above method.